src/js/test/supervisors/Repository.test.ts

```diff
@@ -17,14 +17,37 @@ import { ProcessBatchServer } from "../../modules/rpc/server";
 import { createRecordBatch } from "../../modules/public";
 import { RecordBatch } from "../../modules/public/Coprocessor";
 import sinon = require("sinon");
+import LogService from "../../modules/utilities/Logging";
+import { createSandbox, SinonSandbox } from "sinon";
+import { Script_ManagerServer as ManagementServer } from "../../modules/rpc/serverAndClients/server";
+
+let sinonInstance: SinonSandbox;
+
+const createSinonInstances = (sinonInstance: SinonSandbox) => {
```

Comment:
This should probably be called `stubLogService`.

Reply:
Umm right now it just mock a LogService, but in this function, I put all stubs like another test.